docs/docs/api-proxy.md

```diff
@@ -19,7 +19,7 @@ module.exports = {
 
 This way, when you `fetch('/api/todos')` in development, the development server
 will recognize that it’s not a static asset, and will proxy your request to
-`http://dev-mysite.com/api/todos` as a fallback.
+`http://dev-mysite.com/api/todos` as a fall back.
```

Comment:
```suggestion
`http://dev-mysite.com/api/todos` as a fallback.
```

"fallback" is correct here! 😄

![image](https://user-images.githubusercontent.com/14703164/47685511-1ac8ea00-dbad-11e8-8b2f-df73a8c4b495.png)